packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Providers/ModelProviders/ModelCustomizationTests.cs

```diff
@@ -31,5 +33,57 @@ public void TestCustomization_CanChangeModelName()
             Assert.AreEqual(customCodeView?.Name, modelTypeProvider.Type.Name);
             Assert.AreEqual(customCodeView?.Type.Namespace, modelTypeProvider.Type.Namespace);
         }
+
+        [TestCase]
+        public async Task TestCustomization_CanChangeToStruct()
```

Comment:
nit: lets drop the method name prefix as they are all in a customization class?  If we want them organized together in in the test viewer we can make a folder under ModelProviders called Customization and have this class be in that namespace?

Reply:
Sure!

Reply:
Fixed